client/api/ui/list.ts

```diff
@@ -34,15 +34,15 @@ function reloadList(params: ICombinedEventOrPlanningSearchParams = {}) {
 
     if (currentView === MAIN.FILTERS.PLANNING) {
         dispatch(actions.eventsPlanning.ui.clearList());
-        dispatch(actions.events.ui.clearList());
+        dispatch(actions.planning.ui.clearList());
```

Comment:
This is invalid. If the user is currently in the Planning view, then we clear other lists (in this code block, that would be combined and planning lists). So the changes in this file should be reverted

Reply:
alright